### PR TITLE
Add future-fstrings recipe

### PR DESCRIPTION
**Pull request by sterlingbaldwin**
Adds a recipe for the future-fstrings package.

### CODE REVIEW

File: recipes/future-fstrings/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  summary:
+    Adds fstrings for python<3.6
+
+  description: |
+    A backport of fstrings to python<3.6.
+
+extra:
+  recipe-maintainers:
+    - sterlingbaldwin
+    - asottile
```
Comment (mariusvniekerk) [inline recipes/future-fstrings/meta.yaml]: @asottile  are you okay with being a maintainer of this package on conda-forge?

**Comment by mariusvniekerk** [comment]
Looks like we need to have `tokenize-rt` first

File: recipes/future-fstrings/meta.yaml (hunk 1 above)
Comment (asottile) [reply recipes/future-fstrings/meta.yaml]: (forgive my ignorance): what does that mean for me?

File: recipes/future-fstrings/meta.yaml [hunk 2]
```diff
@@ -18,8 +18,10 @@ requirements:
   build:
     - python
     - setuptools
+    - tokenize-rt
   run:
     - python
+    - tokenize-rt
```
Comment (asottile) [inline recipes/future-fstrings/meta.yaml]: does conda have a concept of conditional dependencies? `tokenize-rt` is only needed for python<3.6

File: recipes/future-fstrings/meta.yaml [hunk 3]
```diff
@@ -18,8 +18,10 @@ requirements:
   build:
     - python
     - setuptools
+    - tokenize-rt
```
Comment (asottile) [inline recipes/future-fstrings/meta.yaml]: `tokenize-rt` shouldn't be needed at build as far as I know

File: recipes/future-fstrings/meta.yaml (hunk 2 above)
Comment (mariusvniekerk) [reply recipes/future-fstrings/meta.yaml]: You can do this with a selector (`- tokenize-rt  # [py<36]` ), but that means you can't build this with `python: noarch`

File: recipes/future-fstrings/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/future-fstrings/meta.yaml]: Basically we will make a new repository `conda-forge/future-strings-feedstock` that will be used to maintain this package.  Presently we have bots that will actually make pull requests when the pypi version of this package changes and rebuild for the new version.  These pull requests have to be merged by a maintainer or a conda-forge admin.  

For pure python packages the overhead is pretty.  Realistically the only thing that i can think of that could cause an issue for a package of this nature is when py37 rolls out and for some reason it no longer works.

Comment (asottile) [reply recipes/future-fstrings/meta.yaml]: Seems fine, I can press green buttons :)

**Comment by sterlingbaldwin** [comment]
I accidentally merged the tokenize-rt recipe branch into this one, but it should now be fixed. This wont pass the CI tests until tokenize-rt is merged into conda-forge.

**Comment by asottile** [comment]
oh boy was I in for a wild ride, can you switch the maintainer on this to `deadsnakes-issues-bot`?

https://twitter.com/codewithanthony/status/1012942076323852288

**Comment by asottile** [comment]
@sterlingbaldwin while I have ya, can you merge this one for me? https://github.com/conda-forge/tokenize-rt-feedstock/pull/1

**Comment by sterlingbaldwin** [comment]
Doesnt look like I have any control over that repo unfortunately.

**Comment by asottile** [comment]
ah, you should have an organization + team invite in your email for `conda-forge` :)

**Comment by sterlingbaldwin** [comment]
Ah yes, found and merged. Do you know how to fix the circleci problem? It looks like its failing to download tokenize-rt because of a checksum failure, but I dont know where its getting that expected checksum value from.

**Comment by asottile** [comment]
hmm yeah no idea there, maybe one of the more seasoned `conda` devs can chime in?

**Pull request merged**